dlrs/main/classes/RollupCalculateJobSchedulable.cls

```diff
@@ -49,21 +49,37 @@ public with sharing class RollupCalculateJobSchedulable implements Schedulable {
       String jobRef = jobNameByTriggerId.size() == 0
         ? 'id ' + triggerId
         : 'named ' + jobNameByTriggerId.get(triggerId) + ' (' + triggerId + ')';
+
+      // Retrieve organization info to put in the email
+      Organization org = [SELECT Id, Name FROM Organization];
+
+      //Getting rollup name
+      String rollupName = rollup.isEmpty() ? '' : rollup[0].Name;
+
       // Errors are emailed to the user that scheduled this job
       Messaging.SingleEmailMessage mail = new Messaging.SingleEmailMessage();
       mail.setToAddresses(new List<String>{ UserInfo.getUserId() });
       mail.setSubject(
         String.format(
-          'Apex scheduled job {0} failed to run rolllup',
+          'Apex scheduled job "{0}" failed to run rolllup',
```

Comment:
Can you normalize this somewhat with the one used in the other job? I like it having "Scheduled" in the name but I think otherwise it should be identical.